fhir_2_ada-r4/mp/9.2.0/beschikbaarstellen_medicatiegegevens/payload/beschikbaarstellen_medicatiegegevens_2_ada.xsl

```diff
@@ -40,7 +40,10 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
 
 					<xsl:choose>
 						<xsl:when test="count(f:Bundle/f:entry/f:resource/f:Patient) ge 2 or count(distinct-values(f:Bundle/f:entry/f:resource/(f:MedicationRequest | f:MedicationDispense | f:MedicationStatement)/f:subject/f:reference/@value)) ge 2">
-							<xsl:message terminate="yes">Multiple Patients and/or subject references found. Please check.</xsl:message>
+							<xsl:call-template name="util:logMessage">
+								<xsl:with-param name="level" select="$logWARN"/>
```

Comment:
Perhaps these types of messages should be ERROR; see similar messages in e.g. contraindicatie and lab. In the files below this should be changed as well (if we choose to do so).

Reply:
Agree